.github/workflows/ci_push_testing.yml

```diff
@@ -116,13 +110,11 @@ jobs:
     name: GangaDirac Unit
     needs: lint
     runs-on: ubuntu-latest
-    container: centos:7
+    container: centos/python-38-centos7
     steps:
     - uses: actions/checkout@v1
-    - name: Set up Python 3
-      run: yum install -y python3
     - name: Install git
-      run: yum install -y git
+      run: sudo yum install -y git
```

Comment:
Is the `sudo` necessary? It has caused a failure

Reply:
Didn't work without, didn't work with it! I will interactively jump into container and try to work out what the problem is. Even if the python38 container is created by centos, it is a bit different to the standard centos7 container.